openpype/hosts/blender/hooks/pre_pyside_install.py

```diff
@@ -86,7 +86,6 @@ def inner_execute(self):
 
         # Change PYTHONPATH to contain blender's packages as first
         python_paths = [
-            python_lib,
```

Comment:
Why removing this line?

Reply:
Okay, can you add it back, comment it out like `# python_lib,` and add a comment to explain this caused issues because of the multiple versions of Python used, so we keep track of what we disable, because removing it will hide it, if in the future we have issues

Reply:
It is done !